### PR TITLE
Add desktop app blog

### PR DESCRIPTION
**Pull request by resouer**
A blog about how do deploy desktop application on Hyper_ cloud.

### CODE REVIEW

**Comment by resouer** [comment]
@jmknyc06 Please review this when you have time.

File: blog/source/run-desktop-applications-in-hyper_.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+title: "Run Desktop Applications in Hyper_"
+date: 2016-06-07 20:00:00 +0800
+author: hyper
+preview: Hyper_ cloud is designed to work well with many kinds of applications. That's why most people use it to deploy their applications in a cloud environment. This is great, and saves people huge headaches, but this article will show you a not-so-typical way.
+
+---
+
+In this article, we will show how to use Hyper_ to help run desktop applications on the cloud. The basic idea is that we will launch a Linux desktop version OS on Hyper_ cloud and then we can launch Chorme, Firefox etc in that container through VNC.
```
Comment (kevinsimper) [inline blog/source/run-desktop-applications-in-hyper_.md]: Chrome is misspelled here :)

File: blog/source/run-desktop-applications-in-hyper_.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+But why would we even want to run all these applications in Hyper_? 
+The most common case is that users may not want to install all the things on their PCs. 
+They may also want the ability to delete an application and know it is gone without some random file hanging around. 
+
+Obviously, running those applications in Hyper_ cloud will help a lot. What's more, now users can control how much CPU and Memory the application uses, and visit this application anywhere as long as they have Internet. 
+
+Non like sharing kernel in OS containers, Hyper_ leverage independent Linux Kernel for every container, that makes running desktop application in Hyper_ much easier, let's see how to do!
```
Comment (kevinsimper) [inline blog/source/run-desktop-applications-in-hyper_.md]: Should "Non" be "Not" here?

File: blog/source/run-desktop-applications-in-hyper_.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+```shell
+$ fip=`hyper fip allocate 1`
+$ hyper fip associate ${fip} ${containerId}
+```
+
+Done! Now you can use VNC client to connect your CentOS container at  `fip:1`.
+
+### Try to experience more
+
+The most exciting part of "container revolution" is finally we can ship applications to cloud in a elegant way. Thanks to container images! In Hyper_, you can experience more desktop applications by deploying them in one shell, and connect them using `fip` from VPN client.
```
Comment (kevinsimper) [inline blog/source/run-desktop-applications-in-hyper_.md]: "a" should be "an" before elegant :)

File: blog/source/run-desktop-applications-in-hyper_.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+For example, `kaixhin/vnc` is a image which provide you with `Ubuntu Core 14.04 + LXDE desktop + Firefox browser + TightVNC server`. You can deploy it to Hyper_ easily:
+```shell
+$ hyper run -d  kaixhin/vnc
+$ hyper fip associate 162.221.195.27  0649e517b002
+```
+Then you can use this Ubuntu workstation on Hyper_ cloud by conntecting to `162.221.195.27:1` with password "password", and browser Internet with FireFox.
```
Comment (kevinsimper) [inline blog/source/run-desktop-applications-in-hyper_.md]: *connecting

**Comment by kevinsimper** [comment]
This is super interesting as I have used spun up many digitalocean boxes just to start `dorowu/ubuntu-desktop-lxde-vnc` when i was in Peru because downloading large gigabytes files was 100x quicker on a server than on a 1mbit connection.

There were some minor spelling mistakes, but the article is good. I think a link and a mention to a vnc client would be nice, as the people who read this most likely don't have a VNC program or have every used it before!

File: blog/source/run-desktop-applications-in-hyper_.md (hunk 4 above)
Comment (resouer) [inline blog/source/run-desktop-applications-in-hyper_.md]: Thanks, will update this soon :)

**Comment by resouer** [comment]
@kevinsimper  updated the blog :)

File: blog/source/run-desktop-applications-in-hyper_.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
+title: "Run Desktop Applications in Hyper_"
+date: 2016-06-07 20:00:00 +0800
+author: hyper
+preview: Hyper_ cloud is designed to work well with many kinds of applications. That's why most people use it to deploy their applications in a cloud environment. This is great, and saves people huge headaches, but this article will show you a not-so-typical way.
+
+---
+
+In this article, we will show how to use Hyper_ to help run desktop applications on the cloud. The basic idea is that we will launch a Linux desktop version OS on Hyper_ cloud and then we can launch Chrome, Firefox etc in that container by using VNC.
+
+> More about VNC: VNC is a great client/server tool to access your computers remotely. Please check [tigervnc](http://tigervnc.org/) for example if you are not familiar with it.
```
Comment (kevinsimper) [inline blog/source/run-desktop-applications-in-hyper_.md]: 👍

File: blog/source/run-desktop-applications-in-hyper_.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+> More about VNC: VNC is a great client/server tool to access your computers remotely. Please check [tigervnc](http://tigervnc.org/) for example if you are not familiar with it. 
+
+Wait, why would we even want to run all these applications in Hyper_? 
+The most common case is that users may not want to install all the things on their PCs. 
+They may also want the ability to delete an application and know it is gone without some random file hanging around. 
+Or, they may just want a machine lives in the cloud!
+
+Obviously, running those applications in Hyper_ cloud will help a lot. What's more, now users can control how much CPU and Memory the application uses, and visit this application anywhere as long as they have Internet. 
+
+Not like sharing kernel in OS containers, Hyper_ leverage independent Linux Kernel for every container, that makes running desktop application in Hyper_ much easier, let's see how to do!
```
Comment (kevinsimper) [inline blog/source/run-desktop-applications-in-hyper_.md]: I don't think you have to mention that hyper leverages independent linux kernels to get your point across in this post, as it is not clear why it is "easier"? Maybe a link to somewhere else that explains the benefit of individual kernels? 👍

**Comment by kevinsimper** [comment]
Really good! Added a minor comment, nothing big.

File: blog/source/run-desktop-applications-in-hyper_.md (hunk 6 above)
Comment (resouer) [inline blog/source/run-desktop-applications-in-hyper_.md]: Independent kernel seems to be a different story to tell (for example, when your app tries read from `/proc` or tuning kernel), so I will delete this part. Thanks!

**Pull request merged**